mangaki/mangaki/admin.py

```diff
@@ -76,6 +191,10 @@ class WorkAdmin(admin.ModelAdmin):
         'controversy',
     )
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.filter(redirect__isnull=True)  # Exclude phantom works
```

Comment:
Looks like this is indeed only used for one-to-one fields due to historical reasons, which is unfortunate. I think I'd prefer using `WorkCluster.objects.get(id=10).works(manager='all_objects').all()` instead, since `WorkCluster` is "special" in the sense that it essentially the only class that should see the removed `Works`, we want them to appear as non-existent for other objects. Not 100% sure though, I think the manual solution works for now -- I'd appreciate a comment there explaining that we use manual filtering because ManyToManyField doesn't honor the `default_manager`.